### PR TITLE
added Server#motd and BookMeta#pages

### PR DESCRIPTION
**Pull request by Machine-Maker**
relevant to #4830 / #4842

### CODE REVIEW

File: Spigot-API-Patches/0005-Adventure.patch [hunk 1]
```diff
@@ -775,6 +825,34 @@ index f70a6a22b85ff0da76e67e9b223ad4e0b020b5c4..81925276bb056da0d48bea23128ba0e4
...
+      */
+     void setPage(int page, @NotNull String data);
+ 
++    // Paper start
++    /**
++     * Gets all the pages.
++     *
++     * @return list of all the pages in the book
++     */
++    @NotNull List<net.kyori.adventure.text.Component> pages();
```
Comment (Proximyst) [inline Spigot-API-Patches/0005-Adventure.patch:866]: Setters?

File: Spigot-API-Patches/0005-Adventure.patch [hunk 2]
```diff
@@ -318,6 +344,30 @@ index 6cbc22c08ce421bcd8bf3efabc96c9cec447faee..0325e1ea20d346f9e6d1f128dbbe0150
...
+      */
+     boolean isPrimaryThread();
+ 
++    // Paper start
++    /**
++     * Gets the message that is displayed on the server list.
++     *
++     * @return the server's MOTD
++     */
++    @NotNull net.kyori.adventure.text.Component motd();
```
Comment (Proximyst) [inline Spigot-API-Patches/0005-Adventure.patch:357]: Could've sworn there was a setter for String here.

**Review by Proximyst — APPROVED** [review]
Small notes I'm sure can be fixed. LGTM. Assigning kashike and requesting review.

File: Spigot-API-Patches/0005-Adventure.patch (hunk 1 above)
Comment (Machine-Maker) [reply Spigot-API-Patches/0005-Adventure.patch:866]: oh yeah, :facepalm:

**Review by kashike — APPROVED** [review]

**Pull request merged**